Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -859,3 +889,43 @@ end Supp
 
 -- end of section
 end AddValuation
+
+namespace Valuation
+
+
+variable {Γ₀ : Type*} [Ring R] [LinearOrderedCommMonoidWithZero Γ₀]
+
+/-- The `AddValuation` associated to a `Valuation`. -/
+def toAddValuation : Valuation R Γ₀ ≃ AddValuation R (Additive Γ₀)ᵒᵈ :=
+  AddValuation.ofValuation (R := R) (Γ₀ := (Additive Γ₀)ᵒᵈ)
+
+/-- The `Valuation` associated to a `AddValuation`.
+-/
+def ofAddValuation : AddValuation R (Additive Γ₀)ᵒᵈ ≃ Valuation R Γ₀ :=
+  AddValuation.toValuation
```

Comment:
I'm wondering if we need 2 * 2 defs for passing back and forth.

Atm there are no lemmas explaining what happens if I turn a valuation into an additive valuation using `toAddValuation`, and then return back using `toValuation` from the other file.
And undoubtedly that will happen at some point in the future.

Do you think the API needs all 4 defs? or would 1 in either direction be sufficient?

Reply:
```quote
Atm there are no lemmas explaining what happens if I turn a valuation into an additive valuation using toAddValuation, and then return back using toValuation from the other file.
```
I'm not sure I understand — applying this is simplified, and the valuations are defeq but using that might be defeq abuse, so I'm not sure if there should be a lemma for that